spec/features/staff/users_can_create_a_fund_level_activity_spec.rb

```diff
@@ -141,7 +141,7 @@
 
         choose "Primary education"
         click_button I18n.t("form.activity.submit")
-        expect(page).to have_content I18n.t("page_title.activity_form.show.status")
+        expect(page).to have_content I18n.t("activerecord.attributes.activity.status")
```

Comment:
If the page title for status is staying as "Status" then I'd be looking to see why our pape_title code in the wrapper stopped working. If we stop using page title we lose the ability to change this value to something other than the active record attribute name, which removes the previous flexibility.